gui/src/main/java/org/jboss/as/console/mbui/widgets/ComplexAttributeForm.java

```diff
@@ -70,7 +77,17 @@ public ComplexAttributeForm(String attributeName, SecurityContext securityContex
 
     private ResourceDescription getAttributeDescription() {
         ModelNode desc = new ModelNode();
-        desc.get("attributes").set(resourceDescriptionDelegate.get("attributes").get(attributeName).get("value-type"));
+        desc.get(ATTRIBUTES).set(resourceDescriptionDelegate.get(ATTRIBUTES).get(attributeName).get(VALUE_TYPE));
+        boolean nillable = resourceDescriptionDelegate.get(ATTRIBUTES).get(attributeName).hasDefined(NILLABLE) ?
+                resourceDescriptionDelegate.get(ATTRIBUTES).get(attributeName).get(NILLABLE).asBoolean() : false;
+        if (nillable) {
```

Comment:
This is not a general case, to set the nested attributes as required=true if the parent attribute is required=true, one example is subsystem=elytron/key-store resource, the credential-reference is an attribute (required=true) that contains 4 attributes all of them required=false, either the user can set store or clear-text/type/alias. Other example is identity-mapping OBJECT attribute of elytron/ldap-realm resource. That said, I request to remove this modification to ComplexAttributeForm.java. Thanks.